db/seeds.rb

```diff
@@ -38,6 +38,7 @@
   random_date = rand(1.year).seconds.from_now
   FactoryGirl.create(:conference, :in_current_season,
     location: FFaker::Venue.name,
+    region: ["Africa", "South America", "North America", "Europe", "Asia Pacific"].sample,
```

Comment:
Interesting, I did not know `FFaker` does not have continents _(maybe submit a PR and have another easy to grab contribution?)_